custom_components/gardena_smart_system/const.py

```diff
@@ -19,3 +19,5 @@
 ATTR_OPERATING_HOURS = "operating_hours"
 ATTR_LAST_ERROR = "last_error"
 ATTR_LAST_ERROR_CODE = "last_error_code"
```

Comment:
You should also remove this const as it is not used anymore

Reply:
Absolutely! Done.